tests/Oryx.BuildImage.Tests/CommandTests/OryxCommandTest.cs

```diff
@@ -148,7 +148,7 @@ public void BuildImage_CanExec_SingleCommand()
                 result.GetDebugInfo());
         }
 
-        [Fact]
+        [Fact(Skip="Temporarily skipping the test")]
```

Comment:
@qianz2 may we attach this to a story, on why we are skipping, so we don't forget to re-enable later please?

Reply:
Ok, Here's a link to work item for it: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1468775